docs/articles/documentation/test-api/selecting-page-elements/selector-options.md

```diff
@@ -40,16 +69,8 @@ const gridCell = Selector((m, n) => gridRow(m).children[n], {
 });
 ```
 
-## options.boundTestRun
-
-**Type**: Object
-
-If you need to call a selector from a Node.js callback, assign the current
-[test controller](../test-code-structure.md#test-controller) to the `boundTestRun` option.
-
-For details, see [Calling Selectors from Node.js Callbacks](selectors.md#calling-selectors-from-nodejs-callbacks).
-
 ## options.text
+> Important! ***deprecated*** *Will be removed in next releases. Use [withText()](selectors.md#filter-multiple-dom-nodes) to filter elements by text.*
```

Comment:
***Deprecated.*** *Use the [withText()](selectors.md#filter-multiple-dom-nodes) method to filter elements by text content or a regular expression that matches this content. The [options.text](#optionstext) option will be removed in future releases.*